nightfall-deployer/contracts/Shield.sol

```diff
@@ -283,25 +285,25 @@ contract Shield is Stateful, Structures, Config, Key_Registry, ReentrancyGuardUp
         ERCInterface tokenContract = ERCInterface(addr);
 
         if (t.tokenType == TokenType.ERC20) {
-            if (t.tokenId != ZERO) revert('ERC20 deposit should have tokenId equal to ZERO');
+            if (t.tokenId != ZERO) revert("ERC20 deposit should have tokenId equal to ZERO");
             if (t.value >= super.getRestriction(addr))
-                revert('Value is above current restrictions for deposits');
+                revert("Value is above current restrictions for deposits");
             else tokenContract.transferFrom(msg.sender, address(this), uint256(t.value));
```

Comment:
Should this transfer value be checked too?

Reply:
safeTransfer does revert as expected. It only happens in transfer and transfer from.
Yes, I missed this transferFrom as I'm still analysing errors and warnings shown by slither. I've put DNM tag until I finish.